src/valhalla_export_edges.cc

```diff
@@ -158,17 +158,16 @@ int main(int argc, char* argv[]) {
     options.add_options()
       ("h,help", "Print this help message.")
       ("v,version", "Print the version of this software.")
-      ("c,column", "What separator to use between columns [default=\\0].", cxxopts::value<std::string>(column_separator)->default_value("\0"s))
+      ("c,config", "Path to the json configuration file.", cxxopts::value<std::string>())
+      ("i,inline-config", "Inline json config.", cxxopts::value<std::string>())
+      ("x,column", "What separator to use between columns [default=\\0].", cxxopts::value<std::string>(column_separator)->default_value("\0"s))
```

Comment:
not sure anyone exercised this exe in the past years, I felt it's fine to break the args to make it consistent with everything else

Reply:
I occasionally use it, and perhaps it would be helpful to add a note about this breaking change in the CHANGELOG.

Reply:
Oh good to know! Jep will do that